### PR TITLE
 [screensaver.digitalclock] 5.0.4

### PR DESCRIPTION
**Pull request by vdb86**
Added support for skins Ftv, Grid, Horizon and Reestuarized

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: screensaver.digitalclock/addon.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="screensaver.digitalclock"
 	name="Digital Clock Screensaver"
-	version="5.0.3"
+	version="5.0.4"
 	provider-name="Vojislav Vlasic">
 	<requires>
 		<import addon="xbmc.python" version="2.20.0"/>
```
Comment (Lunatixz) [inline screensaver.digitalclock/addon.xml]: Can you bump this up to 2.25.0 Krypton release.

Comment (vdb86) [reply screensaver.digitalclock/addon.xml]: Sure!
Should I send a pull request just for that part, or the whole thing together?

Comment (Lunatixz) [reply screensaver.digitalclock/addon.xml]: sam pr, rebase it as one commit.

**Comment by vdb86** [comment]
I'll need to do this again, I screwed something up (never learned how to use git properly...:( )

**Comment by Lunatixz** [comment]
@vdb86 here is easy rebase directions. http://kodi.wiki/index.php?title=HOW-TO:Update_a_pull_request_to_an_official_addon_repository_on_GitHub&oldid=130230

**Comment by vdb86** [comment]
Thanks for sending this!

**Pull request closed without merging**